Levi9.ERP.Domain/Repositories/ClientRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using Levi9.ERP.Domain.Models;
+using Levi9.ERP.Domain.Models.DTO;
+
+namespace Levi9.ERP.Domain.Repositories
+{
+    public class ClientRepository : IClientRepository
+    {
+        private readonly DataBaseContext _context;
+        private readonly IMapper _mapper;
+
+        public ClientRepository(DataBaseContext _context, IMapper _mapper)
+        {
+            this._context = _context;
```

Comment:
Look at the example for the ClientController.

Reply:
Fixed